src/main/java/eu/darkbot/api/entities/utils/Attackable.java

```diff
@@ -3,6 +3,80 @@
 import eu.darkbot.api.entities.Entity;
 import eu.darkbot.api.objects.Health;
 import eu.darkbot.api.objects.Info;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public interface Attackable extends Entity, Health, Info {
+
+    /**
+     * @return the target of this {@link Attackable} if present else {@code null}
+     */
+    @Nullable
+    Entity getTarget();
```

Comment:
Unsure if this makes sense.
Imagine those mines in GOP, they probably should be Attackable (so that we can shoot them) but they're never gonna have a target or be aiming or shooting anyone.

Maybe need to distinguish between Attackable and Attacker (where they can attack you), ship implementing both